cisco_aci/tests/fixtures/metadata.py

```diff
@@ -0,0 +1,3385 @@
+        ],
+        'index': 'eth1/23',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/24',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/24',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/25',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/25',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/26',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/26',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/27',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/27',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/28',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/28',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/29',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/29',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/30',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/30',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/31',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/31',
+        'mac_address': 'not-applicable',
```

Comment:
Could we have tests with fixtures where both the mac_address and name are set correctly?
Also do you know why there is no `description` field? I would expect it based on the code

Reply:
yes, very good point, i just used the fixture which i assume had a bunch of defaults, i'll update them to be more meaningful!